voluncheer/profiles/test_password_reset.py

```diff
@@ -0,0 +1,100 @@
+from django.test import TestCase
+from django.urls import reverse
+from profiles.models import User
+from django.core import mail
+from django.utils.http import urlsafe_base64_encode
+from django.utils.encoding import force_bytes
+from django.contrib.auth.tokens import default_token_generator
+
+
+class PasswordResetTest(TestCase):
+    def setUp(self):
```

Comment:
should it be set_up instead?

Reply:
It can be but I am not sure if it matters. I can change it if you'd like!

Reply:
`setUp()` is a built-in testing function in Django. Basically, it is used to write data to the db temporarily before each test and clears itself after each test. For example, if you wanted a test user, you would create the user here, and you can rest assured that would you get a fresh user for each test (as to it being modified and passed along from test to test). 

This should remain as `setUp()`.

Reply:
`setUp` is the proper function, it is inherited from `unittest.TestCase` and pre-dates the PEP-8 style of using `lower_snake_case`.